### DISCUSSION TITLE
can t access to the benefits of students

### DISCUSSION THREAD

**Discussion by Raedaffan** [community/community#19751] [GitHub Education]
I sign up to one month (coding) and I didn't get the benefits

**Comment by surajsahani** [comment]
Use this link https://education.github.com/pack
The process is the same for both if your student just submits any valid ID card.
If your educator Process is the same but you need to have a school-issued email address or upload documents that prove your school affiliation. once submitted , it takes 3-7 days to request approval

**Reply by Raedaffan** [reply]
I did that before

**Comment by JhonnyLusonode** [comment]
Hello @Raedaffan there is already a Discussion about in #19050 
Checking it out maybe there are all the answers you need.

If you find this answer useful please consider marking it as "Answered".
(btw, this would help me to get the badge. Thank you! 😁)

**Comment by sonawaneutkarsh** [comment]
make sure you send a picture of your education identity that is **clear**.
